console/src/components/MapCore/MapCore.js

```diff
@@ -560,16 +581,55 @@ class MapCoreBase extends React.Component {
 		delete rest.proposedDestination;
 		delete rest.setDestination;
 		delete rest.skin;
+		delete rest.topLocations;
 		delete rest.updateNavigation;
 		delete rest.viewLockoutDuration;
 		delete rest.zoomToSpeedScaleFactor;
+		const {duration, distance, eta, selectedDestination, selfDriving} = this.state;
+
 		return (
 			<div {...rest} className={classnames(className, css.map)}>
 				{this.message ? <div className={css.message}>{this.message}</div> : null}
-				<nav className={css.tools}>
-					{tools}
-					<ToggleButton alt="Follow" selected={this.state.follow} underline icon="forward" onClick={this.changeFollow} />
-				</nav>
+				<Column className={css.tools} align="stretch space-between">
+					{
+						selfDrivingSelection && <div>
+							<Divider>SELF DRIVING</Divider>
+							<Button small>AUTO</Button>
+							<Button small>MANUAL</Button>
+						</div>
+					}
+					<div>
```

Comment:
This tag needs to be a `Cell` since it's a child of `Column`. Same goes for any Column/Row/Layout, always only Cells as children. (Also line 595 and 617)

Reply:
I'd like to understand this a bit better. Can you elaborate on the value provided by always using `Cell` when it isn't required?

Reply:
While it may not seem necessary in the specific scenario you're seeing at this moment, `Cell` has several properties that allow it, and more specifically, its contents, to behave as expected when the content is too large, has different dimensions, is being aligned, and many other scenarios. `Cell` is the only "approved" child of `Layout` and `Cell` should only be used in a `Layout`. Just because the side effects are not apparent in this scenario, doesn't mean they won't appear. This is how the relationship between `Layout` and `Cell` has always been.

Reply:
Thanks for the explanation, @Djspaceg. I think I understand where you're coming from on this better. I think it would be great future enhancement if we could adapt `Layout` such that `Cell` isn't the only "approved" child. Let's discuss more later!

Reply:
What I'd _rather_ do is provide a way to automatically wrap any child supplied to Layout/Row/Column in a `Cell` automatically, so the reliable relationship is maintained but a level of tedium is removed from the DX side. Ex:
```
<Column>
   <Button />
   <Button />
</Column>
```
is clearer simpler to read than the verbose (existing) alternative:
```
<Column>
   <Cell shrink>
      <Button />
   </Cell>
   <Cell shrink>
      <Button />
   </Cell>
</Column>
```
It could automatically wrap anything that isn't already a Cell, leaving a nice level of customizability and predictable trustworthy behavior. Happy to discuss later.